### DISCUSSION TITLE
switch from UiAutomator2 to espresso -> socket hangup

### DISCUSSION THREAD

**Discussion by RSM-SBA** [appium/appium#15877] [Q&A] [answered]
I have switched my test to espresso but i can' get the driver connected to the espresso server on the emulator.
Has anyone suggestion where i could search ?

 - [Espresso] Waiting up to 150000ms for Espresso server to be online
 - [debug] [WD Proxy] Matched '/status' to command name 'getStatus'
 - [debug] [WD Proxy] Proxying [GET /status] to [GET http://localhost:8403/status] with no body
 - [WD Proxy] socket hang up
 
 I use appium2.0 with the latest espresso driver

**Comment by KazuCocoa** [comment, marked as answer]
Please report it as an issue by filling the template out.
Then, with logcat also may help
